app/generate-badges.py

```diff
@@ -25,6 +25,12 @@
 
 
 def configure_badge_page(badge_page, options):
+    """
+    Configure the badge page according to the page
+    options as passed in the function
+    :param `badge_page` - Badges directory
```

Comment:
Here badge page is a path to a single badge sheet containing multiple badges, not a directory.

Reply:
Yaa i am having confusion in some parameters. Thanks for pointing this out. :+1: